api/db/migrations/20210908085902_add-method-column-in-assessments.js

```diff
@@ -0,0 +1,31 @@
+const TABLE_NAME = 'assessments';
+const COLUMN_NAME = 'method';
+
+exports.up = async function(knex) {
+  await knex.schema.table(TABLE_NAME, (table) => {
+    table.text(COLUMN_NAME);
+  });
+
+  const nbRows = (await knex.raw('select reltuples from pg_class where relname=\'assessments\'')).rows.shift().reltuples;
+  const maxRowCountForUpdate = 650000;
+  const canFillColumn = nbRows < maxRowCountForUpdate;
+
+  if (canFillColumn) {
+    await knex.raw(`UPDATE "assessments" SET "method" =
```

Comment:
Ici aussi on peut réduire l'utilisation de `knex.raw` (et utiliser les constantes définies plus haut) :
```suggestion
    await knex(TABLE_NAME).update({ [COLUMN_NAME]: knex.raw(`
      CASE "type"
        …
      END
    `)});
```